daemon/src/engine/qcommon/qfiles.h

```diff
@@ -575,7 +575,7 @@ typedef struct
 	float  st[ 2 ];
 	float  lightmap[ 2 ];
 	vec3_t normal;
-	byte   color[ 4 ];
+	Color::Color32Bit color;
```

Comment:
This structure is part of the BSP file format, I think it would be best to leave it as an array of bytes since it is read directly from a file. In particular, byte is a POD type by Color32Bit isn't.